common/src/main/templates/io/netty/util/collection/KObjectHashMap.template

```diff
@@ -458,10 +475,113 @@ public class @K@ObjectHashMap<V> implements @K@ObjectMap<V>, Iterable<@K@ObjectM
         }
```

Comment:
line 467: `values[index] = toInternal(oldVal);` do you need the `toInternal` here?